student-store-ui/src/components/App/App.jsx

```diff
@@ -1,21 +1,55 @@
 import * as React from "react"
-import { BrowserRouter } from 'react-router-dom'
+import { BrowserRouter, Route, Routes } from 'react-router-dom'
 import Navbar from "../Navbar/Navbar"
 import Sidebar from "../Sidebar/Sidebar"
 import Home from "../Home/Home"
+import Hero from "../Hero/Hero"
+import SubNavbar from "../SubNavbar/SubNavbar"
+import ProductDetail from "../ProductDetail/ProductDetail"
+import { useEffect, useState } from "react"
+import axios from 'axios'
 import "./App.css"
 
 export default function App() {
+  const url = "https://codepath-store-api.herokuapp.com/store"
```

Comment:
Since this is a constant variable, you should declare it inside this file, but outside of the `App` component